authentication-management.yaml

```diff
@@ -26,13 +32,130 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/JsonVersionNumber'
+  /users/{username}:
```

Comment:
is this really a descriptive route naming btw?

Reply:
No, the name is likely to change in the future